Mage.Sets/src/mage/cards/h/HideousTaskmaster.java

```diff
@@ -0,0 +1,61 @@
+package mage.cards.h;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.constants.SubType;
+import mage.abilities.Ability;
+import mage.abilities.effects.common.CastSourceTriggeredAbility;
+import mage.abilities.effects.common.UntapTargetEffect;
+import mage.abilities.effects.common.continuous.GainAbilityTargetEffect;
+import mage.abilities.effects.common.continuous.GainControlTargetEffect;
+import mage.abilities.keyword.DevoidAbility;
+import mage.abilities.keyword.TrampleAbility;
+import mage.abilities.keyword.HasteAbility;
+import mage.abilities.keyword.AnnihilatorAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.target.TargetPermanent;
+import mage.target.common.TargetCreaturePermanent;
+import mage.target.targetadjustment.EachOpponentPermanentTargetsAdjuster;
+import mage.target.targetpointer.EachTargetPointer;
+
+
+/**
+ *
+ * @author anonymous
```

Comment:
You can set authorship if you desire so. 

Btw you can set up a author file in the Utils folder to automatically set authorship of generated files.